src/agent/rustjail/src/mount.rs

```diff
@@ -1185,7 +1211,7 @@ mod tests {
         let mounts = HashMap::new();
 
         // there is no spec.linux, should fail
-        let ret = init_rootfs(stdout_fd, &spec, &cpath, &mounts, true);
+        let ret = init_rootfs(stdout_fd, &spec, &cpath, &mounts, true, false);
```

Comment:
Could we add a test case where `expose_configfs` is true?

Reply:
Done!